components/brave_wallet_ui/panel/async/wallet_panel_async_handler.ts

```diff
@@ -467,6 +467,14 @@ handler.on(PanelActions.expandWalletAccounts.getType(), async (store) => {
   })
 })
 
+handler.on(PanelActions.expandWalletAddAsset.getType(), async (store) => {
+  chrome.tabs.create({ url: 'chrome://wallet/crypto/portfolio/add-asset' }, () => {
```

Comment:
Is there a constant for this route?

Reply:
There is a partial constant for these routes that doesn't include `chrome://wallet` 
Updated it to use the second half, as well as others.